tests/Phinx/Db/Adapter/SQLiteAdapterTest.php

```diff
@@ -1898,4 +1898,134 @@ public function provideTablesForTruncation()
             ['create table t(id integer primary key autoincrement)', 'T', 't'],
         ];
     }
+
+    public function testForeignKeyReferenceCorrectAfterRenameColumn()
+    {
+        $refTable = new \Phinx\Db\Table('ref_table', [], $this->adapter);
+        $refTable->addColumn('columnToRename', 'string')->save();
+
+        $table = new \Phinx\Db\Table('table', [], $this->adapter);
+        $table->addColumn('ref_table_id', 'integer');
+        $table->addForeignKey('ref_table_id', 'ref_table', 'id');
+        $table->save();
+
+        $refTable->renameColumn('columnToRename', 'renamedColumn')->save();
+
+        $this->assertTrue($this->adapter->hasForeignKey($table->getName(), ['ref_table_id']));
+        $this->assertFalse($this->adapter->hasTable("tmp_{$refTable->getName()}"));
+        $this->assertTrue($this->adapter->hasColumn($refTable->getName(), 'renamedColumn'));
+
+        $rows = $this->adapter->fetchAll('select * from sqlite_master where `type` = \'table\'');
+        foreach ($rows as $row) {
+            if ($row['tbl_name'] == $table->getName()) {
+                $sql = $row['sql'];
+            }
+        }
+        $this->assertRegExp('/REFERENCES `ref_table` \(`id`\)/', $sql);
+    }
+
+    public function testForeignKeyReferenceCorrectAfterChangeColumn()
+    {
+        $refTable = new \Phinx\Db\Table('ref_table', [], $this->adapter);
+        $refTable->addColumn('columnToChange', 'string')->save();
+
+        $table = new \Phinx\Db\Table('table', [], $this->adapter);
```

Comment:
Also import here.

Reply:
Please don't spam the thread with all instances of the same thing, a "see below for other occurances" usually suffices.